Solutions/Windows Security Events/Hunting Queries/HostsWithNewLogons.yaml

```diff
@@ -43,5 +42,5 @@ query: |
   | where ActionType == 'Logon'
   | summarize count() by ComputerName, AccountName
   ) on ComputerName, AccountName
-  | summarize StartTimeUtc = min(StartTimeUtc), EndTimeUtc = max(EndTimeUtc), HostCount=dcount(ComputerName), HostSet=makeset(ComputerName, 10)  by AccountName, ComputerName
-  | extend timestamp = StartTimeUtc, AccountCustomEntity = AccountName
+  | summarize StartTimeUtc = min(StartTimeUtc), EndTimeUtc = max(EndTimeUtc), HostCount=dcount(ComputerName), HostSet=make_set(ComputerName, 10)  by AccountName, ComputerName
+  | extend timestamp = StartTimeUtc, AccountCustomEntity = AccountName
```

Comment:
We should add entity mappings if we are updating this query

Reply:
done